Source/WebCore/html/HTMLMetaElement.cpp

```diff
@@ -160,32 +161,41 @@ void HTMLMetaElement::process()
     if (!isInDocumentTree())
         return;
 
+    // Web-Specification: https://html.spec.whatwg.org/multipage/semantics.html#the-meta-element
+    // All below situations require a content attribute (which can be the empty string).
     const AtomString& contentValue = attributeWithoutSynchronization(contentAttr);
     if (contentValue.isNull())
         return;
+    
+    const AtomString& httpEquivValue = attributeWithoutSynchronization(http_equivAttr);
+    // Get the document to process the tag, but only if we're actually part of DOM
+    // tree (changing a meta tag while it's not in the tree shouldn't have any effect
+    // on the document)
+    if (!httpEquivValue.isNull())
+        document().processMetaHttpEquiv(httpEquivValue, contentValue, isDescendantOf(document().head()));
+    
+    const AtomString& nameValue = attributeWithoutSynchronization(nameAttr);
+    if (nameValue.isNull())
+            return;
```

Comment:
nit: this is badly indented.

Reply:
Ops! Sorry! Will fix it and push new commit.